webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -195,6 +206,7 @@ export default {
       contentMin: 3,
       hashtags: [],
       elem: null,
+      isCropInProgress: null,
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T19:48:50Z" title="Friday, December 20th 2019, 8:48:50 pm +01:00">Dec 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -264,6 +271,11 @@ export default {
     categoryIds(categories) {
       return categories.map(c => c.id)
     },
+    deleteImage() {
+      this.form.image = ''
+      this.contribution.image = ''
+      this.showDeleteButton = ''
```


Could we set these to proper `null` values?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-06T09:58:27Z" title="Monday, January 6th 2020, 10:58:27 am +01:00">Jan 6, 2020</time>_

---


ok